README.md

```diff
@@ -396,29 +406,46 @@ Combination functions return an array of TinyColor objects unless otherwise note
 
 ### random
 
-Returns a random color
+Returns a random color.
 ```js
 var color = tinycolor.random();
 color.toRgb(); // "{r: 145, g: 40, b: 198, a: 1}"
 ```
 
-### readability
+### Readability
+TinyColor assesses readability based on the [Web Content Accessibility Guidelines (Version 2.0)](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#contrast-ratiodef).
 
-`readable: function(TinyColor, TinyColor) -> Object`.  Analyze 2 colors and returns an object with the following properties.  `brightness` is difference in brightness between the two colors.  `color`: difference in color/hue between the two colors.
+#### readability
 
-    tinycolor.readability("#000", "#111"); // {brightness: 17, color: 51}
-    tinycolor.readability("#000", "#fff"); // {brightness: 255, color: 765}
+`readability: function(TinyColor, TinyColor) -> Object`.
+Returns the contrast ratio between two colors.
 
-### isReadable
+    tinycolor.readability("#000", "#000"); // 1
+    tinycolor.readability("#000", "#111"); // 1.1121078324840545
+    tinycolor.readability("#000", "#fff"); // 21
 
-`isReadable: function(TinyColor, TinyColor) -> Boolean`.  Ensure that foreground and background color combinations provide sufficient contrast.
+Use the values in your own calculations, or use one of the convenience functions below.
 
-    tinycolor.isReadable("#000", "#111"); // false
+#### isReadable
 
-### mostReadable
+`isReadable: function(TinyColor, TinyColor, Object) -> Boolean`.  Ensure that foreground and background color
+combinations meet WCAG guidelines. `Object` is optional, defaulting to defaults to `{level:"AA",size:"small"}`.
 
+    tinycolor.isReadable("#000", "#111", {}); // false
+    tinycolor.isReadable("#ff0088", "#5c1a72",{level:"AA",size:"small"}); //false
+    tinycolor.isReadable("#ff0088", "#5c1a72",{level:"AA",size:"large"}), //true
+
+#### mostReadable
+
+`mostReadable: function(TinyColor, [TinyColor, Tinycolor ...], Object) -> Boolean`.
 Given a base color and a list of possible foreground or background colors for that base, returns the most readable color.
+If none of the colors in the list is readable, `mostReadable` will return the better of black or white if `checkReadability:true`.
```

Comment:
I'm not sure about this parameter name `checkReadability`, it doesn't seem to describe what it is doing.  Seems this is really about always returning an acceptable color, so maybe something like `includeFallbackColors` or `useBlackOrWhiteIfNecessary`?  Not really sure that either of those is a "winner", but what are your thoughts?